src/libraries/System.Net.Http/src/System/Net/Http/HttpClientHandler.AnyMobile.cs

```diff
@@ -424,7 +425,15 @@ private object InvokeNativeHandlerMethod(string name, params object?[] parameter
                 s_cachedMethods[name] = method;
             }
 
-            return method!.Invoke(_nativeHandler, parameters)!;
+            try
+            {
+                return method!.Invoke(_nativeHandler, parameters)!;
+            }
+            catch (TargetInvocationException e)
+            {
+                ExceptionDispatchInfo.Capture(e.InnerException!).Throw();
+                throw;
```

Comment:
Redundant? Wouldn't the line above already throw?

Reply:
It's needed to trick the analyzer.